version/version.go

```diff
@@ -27,6 +19,10 @@ var (
 	BlockProtocol uint64 = 11
 )
 
+// TMVersionDefault is the used as the fallback version of Tendermint Core
+// when not using git describe. It is formatted with semantic versioning.
+const TMVersionDefault = "0.34.11"
```

Comment:
I believe this should be prefixed with `v`, as it is otherwise set as the output of `git describe`

```
git describe
v0.34.12
```